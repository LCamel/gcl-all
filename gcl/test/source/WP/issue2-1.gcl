con A, B : Int
var x, y, z : Int
{ z = A * B }