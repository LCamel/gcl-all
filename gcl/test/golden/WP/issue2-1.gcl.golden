### Result

[ Assertion at ./test/source/WP/issue2-1.gcl [33-46] 3:1-14 #479deb1e3916ef60
`True
â‡’
z = A * B`
explanation not available ]

### Source

con A, B : Int
var x, y, z : Int
{ z = A * B }

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
